CONTRIBUTING.md

```diff
@@ -264,6 +264,51 @@ If you make modifications, they'll be applied automatically after a Webpack rebu
 By default the showcase depends on published Stark packages over at npm.
 But once the main build script at Stark's root (cfr previous section) is executed (e.g., using `npm install` or `npm run build` at the root), then the showcase will depend on the contents of the dist folder.
 
+#### Modifying the news html file in the showcase
+
+To display news about **new releases of Stark**, a static html file, `new.component.html` can be modified.
+That file can be found at the following location:
+
+```txt
++---showcase
+|	+---src
+	|	+---app
+		|	+---news
+			|	+---news.component.html
```

Comment:
The spacing in these lines looks a bit weird, you can replace the tabs by spaces, like we did in the ENVIRONMENTS.md ;)